contrib/signet/miner

```diff
@@ -97,10 +100,7 @@ def finish_block(block, signet_solution, grind_cmd):
         block.rehash()
     return block
 
-def generate_psbt(tmpl, reward_spk, *, blocktime=None, poolid=None):
-    signet_spk = tmpl["signet_challenge"]
-    signet_spk_bin = bytes.fromhex(signet_spk)
-
+def new_block(tmpl, reward_spk, blocktime=None, poolid=None):
```

Comment:
Should keep the `, *,` to ensure `blocktime` and `poolid` are keyword-only args.